packages/yum/test.sh

```diff
@@ -91,7 +91,12 @@ pgroonga_package=$(basename $(ls ${packages_dir}/*-pgroonga-*.rpm | head -n1) |
                      sed -e 's/-pgroonga-.*$/-pgroonga/g')
 postgresql_version=$(echo ${pgroonga_package} | grep -E -o '[0-9.]+')
 
-${DNF} install -y postgresql${postgresql_version}-contrib
+if [ "${os}" = "almalinux" ] || [ ${major_version} = "10" ]; then
```

Comment:
Could you add a comment when we can remove this workaround?

Reply:
I added a comment.